src/API/useDownloadLinks.js

```diff
@@ -17,3 +16,22 @@ export const useDownloadLinks = (id) => {
     data,
   };
 };
+
+export const useProgressStatus = (id, refetchInterval, setRefetchInerval) => {
```

Comment:
`refetchInterval, setRefetchInerval` should be used in this hook, and we just need to put initial value for interval

Reply:
Moved inside the hook useState logic